x/concentrated-liquidity/keeper_test.go

```diff
@@ -33,6 +33,7 @@ var (
 	DefaultZeroSwapFee                             = sdk.ZeroDec()
 	DefaultFeeAccumCoins                           = sdk.NewDecCoins(sdk.NewDecCoin("foo", sdk.NewInt(50)))
 	DefaultPositionId                              = uint64(1)
+	DefaultUnderlyingLockId                        = uint64(0)
```

Comment:
0 is safe to use as a no op identifier for locks. This is because even if we start on a fresh genesis, LastLockId is set to 0, and the first CreateLock that gets called increases this by one before assigning that value to the first lock.